languagetool-language-modules/ar/src/main/java/org/languagetool/language/Arabic.java

```diff
@@ -0,0 +1,117 @@
+/* LanguageTool, a natural language style checker 
+ * Copyright (C) 2014 Daniel Naber (http://www.danielnaber.de)
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.language;
+
+import org.languagetool.Language;
+import org.languagetool.rules.*;
+import org.languagetool.rules.ar.ArabicCommaWhitespaceRule;
+import org.languagetool.rules.ar.ArabicContractionSpellingRule;
+import org.languagetool.rules.ar.ArabicDoublePunctuationRule;
+import org.languagetool.rules.ar.ArabicLongSentenceRule;
+import org.languagetool.rules.ar.ArabicWordRepeatRule;
+import org.languagetool.rules.spelling.hunspell.HunspellNoSuggestionRule;
+import org.languagetool.tagging.Tagger;
+import org.languagetool.tagging.ar.ArabicTagger;
+import org.languagetool.tokenizers.SentenceTokenizer;
+import org.languagetool.tokenizers.WordTokenizer;
+import org.languagetool.tokenizers.ArabicSentenceTokenizer;
+import org.languagetool.tokenizers.ArabicWordTokenizer;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
+/**
+ * Support for Arabic.
+ */
+public class Arabic extends Language {
+
+  private SentenceTokenizer sentenceTokenizer;
+  private WordTokenizer wordTokenizer;
+  private Tagger tagger;
+
+  @Override
+  public String getName() {
+    return "Arabic";
+  }
+
+  @Override
+  public String getShortCode() {
+    return "ar";
+  }
+
+  @Override
+  public String[] getCountries() {
+    return new String[]{"DZ"};
```

Comment:
The tests fail when adding other countries to this file : 

```
2017-04-04 12:51:07 An error has occurred, sending HTTP code 500. Access from 127.0.0.1, HTTP user agent: Java/1.8.0_121, language: en, text length: 7, Stacktrace follows:
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: 'ar-DZ' is not a language code known to LanguageTool. Supported language codes are: ar, ast-ES, be-BY, br-FR, ca-ES, ca-ES-valencia, da-DK, de, de-AT, de-CH, de-DE, de-DE-x-simple-language, el-GR, en, en-AU, en-CA, en-GB, en-NZ, en-US, en-ZA, eo, es, fa, fr, gl-ES, it, ja-JP, km-KH, nl, pl-PL, pt, pt-AO, pt-BR, pt-MZ, pt-PT, ro-RO, ru-RU, sk-SK, sl-SI, sv, ta-IN, tl-PH, uk-UA, xx-XX, zh-CN. The list of languages is read from META-INF/org/languagetool/language-module.properties in the Java classpath. See http://wiki.languagetool.org/java-api for details.
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at org.languagetool.server.TextChecker.checkText(TextChecker.java:127)
	at org.languagetool.server.ApiV2.handleRequest(ApiV2.java:60)
	at org.languagetool.server.LanguageToolHttpHandler.handle(LanguageToolHttpHandler.java:101)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:79)
	at sun.net.httpserver.AuthFilter.doFilter(AuthFilter.java:83)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:82)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(ServerImpl.java:675)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:79)
	at sun.net.httpserver.ServerImpl$Exchange.run(ServerImpl.java:645)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```


Any idea how to fix ? am I missing something ?